scripts/run_pattern_check.py

```diff
@@ -0,0 +1,77 @@
+# coding: utf-8
+#
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Pre-submission script for Oppia.
+# This script checks for unaccepted text patterns in the commit.
+
+import fnmatch
+import os
+import subprocess
+
+_PARENT_DIR = os.path.abspath(os.path.join(os.getcwd(), os.pardir))
+
+
+def _get_changed_filenames():
```

Comment:
Ya, Like we will pass an argument for full check or check only files on this commit.